.github/workflows/jsdocs-main.yml

```diff
@@ -4,7 +4,7 @@ name: JSDocs-Main
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
   push:
-    branches: [main]
+    branches: [ main ]
```

Comment:
redundant changes, intended only to trigger the pipeline with a fresh pull request.